ttnn/cpp/ttnn/operations/pool/avgpool/avg_pool_pybind.hpp

```diff
@@ -8,14 +8,14 @@
 #include <pybind11/stl.h>
 
 #include "ttnn/cpp/pybind11/decorators.hpp"
-#include "ttnn/operations/pool.hpp"
+#include "ttnn/operations/pool/avgpool/avg_pool.hpp"
 #include "ttnn/types.hpp"
 
 namespace py = pybind11;
-
+using array2_t = std::array<uint32_t, 2>;
```

Comment:
Being defined in a header, this spills into the global scope. Can we remove this?

Reply:
Removed.